schema.sql

```diff
@@ -57,9 +57,10 @@ CREATE TABLE specializations (
 );
 
 CREATE TABLE visits (
-    vet_id INT,
     animal_id INT,
+    vet_id INT,
     visit_date DATE,
-    FOREIGN KEY (vet_id) REFERENCES vets(id),
-    FOREIGN KEY (animal_id) REFERENCES animals(id) 
+    PRIMARY KEY (animal_id, vet_id, visit_date),
+    FOREIGN KEY (animal_id) REFERENCES animals (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id)
 );
```

Comment:
- [ ] good job making the performance better, however, adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems please try it out before requesting another review.

Reply:
- [ ] As requested by the previous reviewer, kindly add indexes for FOREIGN KEYS on columns frequently used as this is the expected solution. 

e.g.
``` 
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);